pkgs/top-level/linux-kernels.nix

```diff
@@ -158,7 +158,6 @@ in {
         kernelPatches.bridge_stp_helper
         kernelPatches.request_key_helper
         kernelPatches.fix-em-ice-bonding
-        kernelPatches.CVE-2023-32233
```

Comment:
Aren't we effectively make the hardened variants vulnerable again? These are AFAICS one patch-level behind upstream.

Reply:
Yeah, seems so. :S

Reply:
Preparing a fix currently.

Reply:
Pushed a fix that just adds the patch to the hardened kernels. The other kernels are up-to-date and linux-rt (haven't checked their state) uses its own expression.